docs/platforms/flutter/integrations/routing-instrumentation.mdx

```diff
@@ -107,6 +100,62 @@ final GoRouter _router = GoRouter(
 );
 ```
 
+## Time To Initial Display
+
+Time to initial display (TTID) provides insight into how long it takes your Widget to launch and draw their first frame, by adding a span for navigation to a Widget. The SDK sets the span operation to `ui.load.initial-display` and the span description to the Widget's route name followed by initial display, for example, `MyWidget initial display`.
+There are two ways to set up TTID:
+
+### Approximation
+
+The time to initial display is enabled by default and provides an approximation of TTID.
+
+### Manual
+
+The manual approach allows you to set a more accurate time to initial display manually.
+In order to report TTID manually wrap the desired widget with `SentryDisplayWidget`.
+
+```dart
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+SentryDisplayWidget(
+  child: MyWidget(),
+)
+```
+
+### Combining both approaches
+
+Since the approximation is enabled by default, you can combine both approaches to report the TTID of only certain Widgets manually where you need accurate measurements and leave the rest to be reported automatically via approximation.
```

Comment:
```suggestion
Since the approximation is enabled by default, you can combine both approaches by manually calculating TTID for certain Widgets that need more accurate measurements and leave the rest to be automatically approximated.
```